roles/ceph-handler/templates/restart_rgw_daemon.sh.j2

```diff
@@ -3,51 +3,48 @@
 RETRIES="{{ handler_health_rgw_check_retries }}"
 DELAY="{{ handler_health_rgw_check_delay }}"
```

Comment:
How is this suppose to work on when you're using the same host with multiple instance name ?

Because with the `radosgw_num_instances` we were able to loop over the list but now there will be only one rgw restart script for multiple rgw instance with different value.

At the end the last instance will override the restart script for all the other instances.